docs/unified/clevo/post-install.md

```diff
@@ -59,29 +59,6 @@ and reboot to use the new kernel.
 
 ### Nvidia drivers
 
-> It is only necessary to follow this step if your device has Nvidia GPU
```

Comment:
It's true these steps are already present in generic test setup, but please note that this document is for end users, who aren't likely to read the test setup documentation. Please revert

Reply:
That's true. Thank you. https://github.com/Dasharo/docs/compare/4b8a420..e16ef29